Samples/Samples/Behaviors/BehaviorBase.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+/*
```

Comment:
Just making sure that we follow their style of comments - I rarely see the usage of `/* */`. Should this be also at placed at the top of the file? I'm thinking about something like this https://github.com/xamarinhq/xamu-infrastructure/blob/598f97d6f5cc68fc603873e913ccc9089dbc60df/src/XamU.Core/Collections/OptimizedObservableCollection.cs